mkosi/resources/mkosi-initrd/mkosi.conf.d/10-serpentos.conf

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[Match]
+Distribution=serpentos
+
+[Content]
+# Completely override the default Packages
+Packages=
```

Comment:
We shouldn't override the list of packages but instead move packages that aren't available on SerpentOS to the distribution specific configuration files